.github/ISSUE_TEMPLATE/bugreport.yml

```diff
@@ -39,6 +39,24 @@ body:
         - [Minimal Complete Verifiable Examples](https://stackoverflow.com/help/mcve)
         - [Craft Minimal Bug Reports](https://matthewrocklin.com/minimal-bug-reports)
 
+        Consider listing dependencies in [inline script metadata](https://packaging.python.org/en/latest/specifications/inline-script-metadata/#example).
```

Comment:
how reasonable is someone writing out all the dependencies? seems like this is actually quite hard if doing it manually?

how about having `xr.show_versions()` output the header? then we have a standard output form that people can copy in? 

(doesn't need to be _perfect_, so fine if there's a simpler way, but it should be feasible...)

Reply:
Ahh this is unclear. It's meant to be any dependencies needed for the minimal example. not all versions of xarray.

So in an ideal world it would be xarray, or if there is an interaction with zarr then it might also include zarr. Does that make sense? Would there be a way to prhase that so it's clearer?


See for example: https://github.com/zarr-developers/zarr-python/issues/3427